jdi-light/pom.xml

```diff
@@ -95,13 +95,13 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
-            <version>1.10.19</version>
+            <version>1.8.4</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.14.3</version>
+            <version>7.0.0-beta1</version>
```

Comment:
it's not a good idea to use beta version

Reply:
Reverted back to 6.14.3